packages/braid-design-system/lib/components/Autosuggest/Autosuggest.tsx

```diff
@@ -292,11 +300,43 @@ export type AutosuggestLabelProps = FieldLabelVariant;
 export type AutosuggestProps<Value> = AutosuggestBaseProps<Value> &
   AutosuggestLabelProps;
 
+function normaliseNoSuggestionMessage<Value>(
+  noSuggestionsMessage: AutosuggestBaseProps<Value>['noSuggestionsMessage'],
+  suggestionProp: AutosuggestBaseProps<Value>['suggestions'],
+) {
+  if (noSuggestionsMessage) {
+    return typeof noSuggestionsMessage === 'string'
+      ? { text: noSuggestionsMessage }
+      : noSuggestionsMessage;
+  }
+
+  if ('message' in suggestionProp) {
+    const message = suggestionProp.message;
+    if (process.env.NODE_ENV !== 'production') {
+      // eslint-disable-next-line no-console
+      console.warn(
+        dedent`
```

Comment:
We definitely need something to make this easier haha. This is way too much effort.